docker-compose.yml

```diff
@@ -26,8 +26,9 @@ services:
     build: 
       context: .
     restart: always
-    ports:
-      - 7001:7001
+    # disable port binding disable in host mode
```

Comment:
If not needed remove it.

Reply:
This will be needed later for deployment